internal/services/controller/controller.go

```diff
@@ -290,6 +296,32 @@ func (c *Controller) Run(ctx context.Context) error {
 		}, c.cfg.Interval, ctx.Done())
 	}()
 
+	metricsType := reflect.TypeOf(&v1beta1.PodMetrics{})
+	go func() {
+		const fetchInterval = 30 * time.Second
+		backoff := wait.NewExponentialBackoffManager(fetchInterval, 5*time.Minute, fetchInterval, 2, 0.2, clock.RealClock{})
+		wait.BackoffUntil(func() {
+			result, err := c.metricsClient.MetricsV1beta1().
+				PodMetricses("all-namespaces").
```

Comment:
does this actually work? While I was testing all namespaces were returned only when empty string was provided

Reply:
You're 100% correct, I replaced this according to the metrics-server documentation and forgot to check if it actually works :|